### PR TITLE
Update internal testnet commit hash

### PR DESCRIPTION
**Pull request by adamrturman**
Updated to match master

### CODE REVIEW

File: ci/env/kava-internal-testnet/KAVA.VERSION [hunk 1]
```diff
@@ -1 +1 @@
-357ec96338a4c4fa4624560f1182bf85cc068009
+dc1f8eb1535d985212ae484baff141c854dd2b36
```
Comment (adamrturman) [inline ci/env/kava-internal-testnet/KAVA.VERSION:1]:
```suggestion
fd83da7a05d4b9eed9d482605e077b42c719ad6f
```

File: ci/env/kava-internal-testnet/KAVA.VERSION [hunk 2]
```diff
@@ -1 +1 @@
-357ec96338a4c4fa4624560f1182bf85cc068009
+fd83da7a05d4b9eed9d482605e077b42c719ad6f
```
Comment (adamrturman) [inline ci/env/kava-internal-testnet/KAVA.VERSION:1]:
```suggestion
master
```

**Comment by galxy25** [comment]
@adamrturman approved pending update to internal testnet genesis (apply all the changes made to protonet from commits `646e3766986703e01a03431865226db7ca629514` to `95d9fb9aca30c4cb30fa553f87df1f99a8f34be1` https://github.com/Kava-Labs/kava/commits/master/ci/env/kava-protonet/genesis.json)

https://github.com/Kava-Labs/kava/actions/runs/5147087064/jobs/9267032471?pr=1611

**Comment by adamrturman** [comment]
> @adamrturman approved pending update to internal testnet genesis (apply all the changes made to protonet from commits `646e3766986703e01a03431865226db7ca629514` to `95d9fb9aca30c4cb30fa553f87df1f99a8f34be1` https://github.com/Kava-Labs/kava/commits/master/ci/env/kava-protonet/genesis.json)
> 
> https://github.com/Kava-Labs/kava/actions/runs/5147087064/jobs/9267032471?pr=1611

Is it okay to simply use the protonet genesis for internal testnet?

**Comment by adamrturman** [comment]
> @adamrturman the two testnets have different validator sets, so updating is not a straightforward as copying the protonet genesis file.
> 
> i've added the missing fields for you and updated the authz grants to match a post-kava 13 chain.

Much appreciated @pirtleshell

**Pull request merged**